client/client_test.go

```diff
@@ -122,15 +122,16 @@ func TestCanPerformStartTLS(t *testing.T) {
 		},
 	}
 	for _, tt := range tests {
```

Comment:
If you're looking for consistency, I'd suggest changing 
```
for _, tt := range tests {
```
for 
```
for _, scenario := range scenarios {
```
as it's what I name test cases in most Gatus tests

Reply:
@TwiN Done [`0ae4b63` (#664)](https://github.com/TwiN/gatus/pull/664/commits/0ae4b63c4212f47cdb758077be479e8a7a139788)